lib/eventlog/src/evtrec.c

```diff
@@ -74,6 +74,7 @@ evt_rec_add_tags(EVTREC *e, EVTTAG *first, ...)
   va_list ap;
 
   va_start(ap, first);
+  evt_rec_add_tag(e, first);
```

Comment:
Can you check the users of this function? For example this should be deleted after the patch:
https://github.com/balabit/syslog-ng/blob/c71f47dfca3316398c833d57253d0514e5995a91/lib/messages.c#L210

Reply:
Sure there is no user, but only unit test. (Which does not really test much sadly.)
I think the whole test should be redesigned for eventlog, and because of that it should be done when it is converted to criterion.

Reply:
Overlook from my side, the caller is evt_rec_add_tagsv instead of evt_rec_add_tags